src/commands/wiki.ts

```diff
@@ -0,0 +1,37 @@
+import Discord from 'discord.js';
+import { Command } from '../types';
+import fetch from 'node-fetch';
+
+const wiki: Command = {
+  name: 'wiki',
+  description: 'Zwraca pierwszy wynik wyszukiwania w wikipedii',
```

Comment:
wyszukiwania z wikipedii zamiast "w"

Reply:
pisałem to na szybko, skupiałem się raczej na execute niż reszcie

Reply:
ale to i tak ma sens, bo wyszukuje "w wikipedii", nie?

Reply:
wyszukujesz W
ale tutaj bardziej kontekst chyba jest "skąd wyszukuję"